portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -42,13 +42,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       PreparedQuery results = datastore.prepare(query);
 
       ArrayList<Comment> comments = new ArrayList<>();
-        for (Entity entity : results.asIterable()) {
+      for (Entity entity : results.asIterable()) {
           long id = entity.getKey().getId();
```

Comment:
it should be 2-space at each level of indentation, since that's what the rest of this file does.